README.md

```diff
@@ -34,20 +34,11 @@ $ node example/parse.js -x 3 -y 4 -n5 -abc --beep=boop foo bar baz
 	a: true,
 	b: true,
 	c: true,
-	beep: 'boop'
+	beep: 'boop',
+	ding: false
 }
 ```
 
-# security
-
-Previous versions had a prototype pollution bug that could cause privilege
-escalation in some circumstances when handling untrusted user input.
-
-Please use version 1.2.6 or later:
-
-* https://security.snyk.io/vuln/SNYK-JS-MINIMIST-2429795 (version <=1.2.5)
-* https://snyk.io/vuln/SNYK-JS-MINIMIST-559764 (version <=1.2.3)
-
```

Comment:
i agree, this is what `npm deprecate` is for, and this warning shouldn't have ever been needed.